examples-complex/src/test/java/com/jexunit/examples/businesstests/commands/LoadAndCalculateTestCommand.java

```diff
@@ -27,7 +27,7 @@
 	 *            the TestContext to add the loaded entity to
 	 */
 	@TestCommand
-	public void load(@TestParam long id, TestContext context) {
+	public void load(@TestParam("id") long id, TestContext context) {
```

Comment:
There was no value set for the @TestParam-annotation to test resolving the parameter name!
But this is only working by setting the compiler argument "-parameter" like it was set before in the compiler-plugin configuration!

Reply:
sorry, that's right.